packages/transaction-controller/src/types.ts

```diff
@@ -69,9 +69,11 @@ type TransactionMetaBase = {
   isTransfer?: boolean;
 
   /**
-   * Network code as per EIP-155 for this transaction.
+   * Network code as per EIP-155 for this transaction
+   *
+   * @deprecated Use `chainId` instead.
    */
-  networkID?: string;
+  readonly networkID?: string;
```

Comment:
we might be able to remove `networkID` from the type? I see minimal usage of it in mobile. Really only [this one spot](https://github.com/MetaMask/metamask-mobile/blob/4bfb9bd8a6f85070fd3ba30b6d049d1aaac2f877/app/core/NotificationManager.js#L424-L425)

Search: https://github.com/search?q=repo%3AMetaMask%2Fmetamask-mobile+networkID&type=code&p=1

Reply:
Given the scope of this PR already, I'd vote to leave it for now until we can update mobile as needed to rely solely on chain ID.

Reply:
sounds good